src/main/java/com/microsoft/graph/http/GraphError.java

```diff
@@ -86,4 +86,19 @@ protected String transformErrorCodeCase(@Nonnull final String original) {
         Objects.requireNonNull(original, "parameter original cannot be null");
         return CaseFormat.LOWER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, original);
     }
+
+    /**
+     * Makes a deep copy of specified error
+     * @param error The error we wish to copy
+     * @return The copy of the specified error
+     */
+    public static final GraphError graphErrorCopy(GraphError error) {
```

Comment:
I'd argue this should be an instance method on Graph error named copy instead.

Reply:
I wasn't sure if it would be a good idea to instantiate a new GraphError within a GraphError instance, mainly out of personal preference, though if you think it would make more sense I can go ahead and adjust it.